packages/sdk-btc/source/address.factory.ts

```diff
@@ -24,10 +26,20 @@ export class AddressFactory {
 		this.#network = getNetworkConfig(this.configRepository);
 	}
 
+	public bip39(mnemonic: string, options?: Services.IdentityOptions): Services.AddressDataTransferObject {
+		abort_unless(BIP39.validate(mnemonic), "The given value is not BIP39 compliant.");
+
+		return {
+			type: "bip39",
+			address: BaseAddress.fromPassphrase(mnemonic, this.#network),
```

Comment:
@faustbrian I got this from Ark's code, but I guess it should be the same right? Or is there anything btc-specific to be done?

Reply:
This should be fine because the address derivation of ARK is based on the initial way Bitcoin did it.